packages/liquid-html-parser/src/stage-2-ast.ts

```diff
@@ -1974,7 +1974,8 @@ function toHtmlSelfClosingElement(
 }
 
 function toNullableTextNode(node: ConcreteTextNode | null): TextNode | null {
-  if (!node) return null;
+  if (!node || node.value === '') return null;
```

Comment:
@charlespwd I would prefer to update the `ohm` logic, but it wasn't so straightforward since `paramDescription` is always matched.

Wanted to get a sanity check on this decision, or some help editing `ohm`!